pygfx/renderers/wgpu/engine/pipeline.py

```diff
@@ -23,6 +26,10 @@
 SHADER_CACHE = GpuCache("shader_modules")
 PIPELINE_CACHE = GpuCache("pipelines")
 
+PRINT_WGSL_ON_ERROR = os.environ.get(
+    "PYGFX_PRINT_WGSL_ON_COMPILATION_ERROR", "0"
+).lower() not in ["false", "0"]
```

Comment:
lmk what you think of this name and the general env var logic.

Reply:
Looks fine by me